crates/nu-command/src/filters/find.rs

```diff
@@ -130,6 +130,10 @@ impl Command for Find {
         ]
     }
 
+    fn search_terms(&self) -> Vec<&str> {
+        vec!["filter", "regex", "search", "cond", "condition"]
```

Comment:
I think adding the prefix of another search term is redundant.

```suggestion
        vec!["filter", "regex", "search", "condition"]
```